routes/api.php

```diff
@@ -16,3 +16,10 @@
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+
+Route::get('clubs','ClubController@index');
+Route::post('clubs','ClubController@store');
+Route::get('clubs/{id}','ClubController@show');
+Route::put('clubs/{id}','ClubController@update');
```

Comment:
These can be gathered in one line :
`Route:resource('clubs', 'ClubController');`